playwright/_impl/_network.py

```diff
@@ -320,7 +320,6 @@ async def continue_(
         self._check_not_handled()
         self.request._apply_fallback_overrides(overrides)
         await self._internal_continue()
-        self._report_handled(True)
```

Comment:
Updated in 49b6b8e05922860e57f38e89ca97ab3dcffe13a2. NB: We bring back the skipping chaining `done` handlers. (They will be done because they will have been cancelled, so calling set_result would be an invalid state error)